components/brave_wallet_ui/wallet_panel.mojom

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
seems odd to me that this would go in brave_wallet_ui since that appears to all be js/ts code right now. Is there a reason why you put it here?

Reply:
Just because it spits out a js that the pak file in brave_wallet_ui includes.

Reply:
or rather that the webpak bundle includes in its grd and then which is included in a pak file